pkg/cmd/playback.go

```diff
@@ -0,0 +1,245 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"github.com/stripe/stripe-cli/pkg/playback"
+	"github.com/stripe/stripe-cli/pkg/validators"
+)
+
+const defaultPort = 13111
+const defaultWebhookPort = 13112
+const endpointsDocString = `
+--- Controlling the server ---
+You can configure the running server instance via HTTP GET endpoints (prefixed with "/pb/").
+
+--- List of server control endpoints ---
+GET /pb/mode/[mode]
+Sets the server mode to one of ["auto", "record", "replay"]
+
+GET /pb/cassette/setroot?dir=[path_to_directory]
+Set the root directory for reading/writing cassettes. All cassette paths are relative to this directory.
+
+GET /pb/cassette/load?filepath=[filepath]
+Load the cassette file at the given filepath, relative to the cassette root directory.
+
+GET /pb/cassette/eject
+Eject (unload) the current cassette and do any teardown. In record mode, this writes the recorded interactions to the cassette file.
+`
+
+type playbackCmd struct {
+	cmd *cobra.Command
+
+	mode string
+
+	apiBaseURL     string
+	filepath       string
+	cassetteDir    string
+	address        string
+	webhookAddress string
+	serveHTTPS     bool
+}
+
+func newPlaybackCmd() *playbackCmd {
+	pc := &playbackCmd{}
+
+	pc.cmd = &cobra.Command{
+		// This flag doesn't actually hide the command because we use a template to display the --help text
+		// To unhide this command, see getUsageTemplate() in templates.go, where we explicitly hide the `playback` cmd.
```

Comment:
```suggestion
		// This flag doesn't actually hide the command because we use a template to display the --help text
		// To unhide this command, see getUsageTemplate() in templates.go, where we explicitly hide the `playback` cmd.
```
If I understand correctly, doesn't the change you made to templates.go *cause* the .Hidden flag to hide the command? i.e. if I added Hidden to `stripe listen`, it would be hidden without me having to make another change in templates.go?

If that's the case, we should remove the comment.

Reply:
Ah true, good catch.